global-news-app/src/components/Button/Button.tsx

```diff
@@ -0,0 +1,19 @@
+import { ButtonProps } from "@mui/material";
+import { StyledButton } from "./button.styled";
+import ArrowRight from "assets/icons/ArrowButton";
+
+export interface IButton extends ButtonProps {
+  text: string;
+  buttontype: "primary" | "secondary" | "text";
+  icon?: boolean;
+}
+
+const Button = ({ text, icon, ...rest }: IButton) => {
+  return (
+    <StyledButton endIcon={icon && <ArrowRight />} {...rest}>
```

Comment:
ideally based on the prop the component should dynamically render the relevant icon

Reply:
At first I did as you suggest but I changed because the storybook didn't work with it

Reply:
Yes, for the purpose of the task the only icon is fine, but would be something nice to have in general, not in priority now.